docs/topics/apple-framework.md

```diff
@@ -4,7 +4,7 @@ Kotlin/Native provides bi-directional interoperability with Objective-C/Swift.
 Objective-C frameworks and libraries can be used in Kotlin code.
 Kotlin modules can be used in Swift/Objective-C code too.
 Besides that, Kotlin/Native has
-[C Interop](https://github.com/JetBrains/kotlin-native/blob/master/INTEROP.md).
+[C Interop](native-c-interop.md).
```

Comment:
I'm not sure how to properly use internal links. Please let me know if this is wrong.

Reply:
Yep, this is correct.